src/Components/Helper/Helper.js

```diff
@@ -0,0 +1,15 @@
+const Actions = require('./Action/Action.json'); 
+export const filterByStatus = (todos = [], status = '', id = '') => {
```

Comment:
The method name should be more objective.

- [ ] rename `filterTodosByStatus `